src/js/components/ModalComponent.jsx

```diff
@@ -51,18 +52,19 @@ var ModalComponent = React.createClass({
   },
 
   render: function () {
-    var modalDialogClassName =
-      "modal-dialog " + modalSizeClassName(this.props.size);
+    var props = this.props;
+    var isInState = this.state.isIn;
```

Comment:
:joy: Yap... It took a while after I understood what it is used for. I kept the naming, as it is the standard bootstrap animation/transition *in* classname. Do you think we should refactor this and introduce our own transition class?